tidb-cloud/integrate-tidbcloud-with-vercel.md

```diff
@@ -145,19 +151,74 @@ The detailed steps are as follows:
+2. Click **Manage** in the TiDB Cloud entry.
+3. Click **Configure**.
+4. Click **Add Link** or **Remove** to add or remove connections.
+
+    ![Vercel Integration Configuration Page](/media/tidb-cloud/vercel/integration-vercel-configuration-page.png)
+
+    When you remove a connection, the environment variables set by the integration workflow are removed from the Vercel project, too. However, this action does not affect the data of the TiDB Serverless cluster.
+
+### Connect with TiDB Serverless branching
+
+Vercel's [Preview Deployments](https://vercel.com/docs/deployments/preview-deployments) feature allows you to preview changes to your app in a live deployment without merging those changes to your Git project's production branch. With [TiDB Serverless Branching](/tidb-cloud/branch-overview.md), you can create a new instance for each branch of your Vercel project. This allows you to preview app changes in a live deployment without affecting your production data.
+
+> **Note:**
+>
+> Currently, TiDB Serverless branching only supports [Vercel projects associated with GitHub repositories](https://vercel.com/docs/deployments/git/vercel-for-github).
+
+To enable TiDB Serverless Branching, you need to:
+
+1. Select **Cluster** as your connection type. 
+2. Enable TiDB Serverless Branching in the TiDB Cloud Vercel integration workflow.
+
+After you push changes to GitHub repository, Vercel will trigger a preview deployment. And then TiDB Cloud integration will automatically create a TiDB Serverless branch for the branch and set environment variables. The detailed steps are as follows:
+
+1. Create a new branch in your Git repository.
+
+    ```shell
+    cd tidb-prisma-vercel-demo1
+    git checkout -b new-branch
+    ```
+
+2. Add some changes and push the changes to the remote repository.
+3. Vercel will trigger a preview deployment for the new branch.
+
+    ![Vercel Preview_Deployment](/media/tidb-cloud/vercel/vercel-preview-deployment.png)
+
+    1. During the deployment, TiDB Cloud integration will automatically create a TiDB Serverless branch with the same name as the git branch. If the TiDB Serverless branch already exists, TiDB Cloud integration will skip this step.
+
+        ![TiDB_Cloud_Branch_Check](/media/tidb-cloud/vercel/tidbcloud-branch-check.png)
+
+    2. After the TiDB Serverless branch is ready, TiDB Cloud integration will set environment variables in the preview deployment for the Vercel project.
+
+        ![Preview_Envs](/media/tidb-cloud/vercel/preview-envs.png)
+
+    3. TiDB Cloud integration will also register a blocking check to wait for the TiDB Serverless branch to be ready. You can rerun the check manually.
+4. After the check is passed, you can visit the preview deployment to see the changes.
+
+> **Note:**
+>
+> Due to the limitation of Vercel deployment workflow, the environment variable can't be ensured to be set in the deployment. Please redeploy if facing this issue.
```

Comment:
```suggestion
> Due to a limitation of the Vercel deployment workflow, the environment variable is not guaranteed to be set in the deployment. In this case, you need to push new changes to trigger a new deployment.
```

Reply:
redeploy is not equal to pushing new changes.